tests/Feature/Booking/BookingTest.php

```diff
@@ -275,85 +280,142 @@ public function testInvalidBookingCantBeCreated()
     }
 
     /** @test */
-    public function testOverlappingBookings()
+    public function testOverlappingBookingsCannotBeCreatedOverlappingOnStart()
```

Comment:
This booking request overlaps on the end

Reply:
Does it? The one in the requests ends after 15:00 (the time in the created).

Reply:
The booking you create is 14:00 to 15:00, and you are trying to make a booking between 14:30 and 15:30. The 14:00 isn't being overlapped by the booking, but the 15:00 (the end) is?